lib/streamlit/config.py

```diff
@@ -358,9 +358,9 @@ def _global_development_mode() -> bool:
     description="""Only cache ForwardMsgs that are greater than or equal to
         this minimum.""",
     visibility="hidden",
-    default_val=10 * 1e3,
+    default_val=int(1e6),
```

Comment:
Surprisingly here, and in `maxMessageSize` we use "megabytes" and "kilobytes" of power of 10 (e.g. 1kb = 1000 bytes, and not 1024 bytes)